apps/bbsync/tests/test_cc.py

```diff
@@ -0,0 +1,809 @@
+import json
+
+import pytest
+
+from apps.bbsync.cc import CCBuilder
+from apps.bbsync.constants import RHSCL_BTS_KEY, USER_BLACKLIST
+from apps.bbsync.tests.factories import BugzillaComponentFactory, BugzillaProductFactory
+from osidb.models import Affect, Flaw, PsModule
+from osidb.tests.factories import (
+    AffectFactory,
+    FlawFactory,
+    PsContactFactory,
+    PsModuleFactory,
+    PsProductFactory,
+    PsUpdateStreamFactory,
+)
+
+pytestmark = pytest.mark.unit
+
+
+class TestCCBuilder:
+    def prepare_flaw(
```

Comment:
I think it makes sense to have this in `conftest.py` and various fixtures built on top of it that are then used in the tests